src/main/java/com/g4s8/ghman/web/TkApp.java

```diff
@@ -88,8 +87,8 @@
                                     final String token = new JdkRequest("https://github.com/login/oauth/access_token")
                                         .method(Request.POST)
                                         .uri()
-                                        .queryParam("client_id", System.getenv("GH_CLIENT"))
-                                        .queryParam("client_secret", System.getenv("GH_SECRET"))
+                                        .queryParam("client_id", environment.getGithubClientId())
```

Comment:
@samihibrahim this line is too long, can we rename `environment` to `env` to make it a bit shorter?

Reply:
@g4s8 Fixed.